packages/manager/modules/sign-up/src/form/details/details.controller.js

```diff
@@ -110,7 +110,8 @@ export default class SignUpDetailsCtrl {
   =            Callbacks            =
   ================================= */
 
-  onCountryChange() {
```

Comment:
As disscussed, we should not need to handle any rebinding on the controller side,
I'm currently checking with Anoop to have an update on the oui-select component to have the binding on the country code working (PR: https://github.com/ovh/ovh-ui-kit/pull/836/).